### DISCUSSION TITLE
Fail to fork the very repo for HTTP 500 error

### DISCUSSION THREAD

**Discussion by SihanLu212** [community/community#47541] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

Hi there,

I'm new to GitHub API and found that I can't fork some repo via API v3, but I can manually fork it. But for other repos, both are okay.

e.g.

```
https://github.com/dgiot/dgiot
```

For the https://github.com/dgiot/dgiot  , the API always returns 500 error.

**Comment by unknowVariable** [comment]
Thank you for reporting this issue with the GitHub API. It's possible that there is an issue with the API forking certain repositories.

To help investigate the issue further, I would recommend providing more details about the specific API request that you are making, including the endpoint and any relevant parameters. Additionally, it would be helpful to include any error messages or response codes that you are receiving, as well as any relevant authentication details.

In the meantime, you can also try reaching out to the GitHub support team directly for assistance with this issue. They may be able to provide more information about the cause of the error and any potential workarounds or solutions.
